### PR TITLE
Update to 1.8rc1

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jakirkham** [comment]
I'm guessing you are referring to this [bug]( https://cmake.org/Bug/view.php?id=13234 ). Hmm...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,7 +21,6 @@ requirements:
   build:
     - python
     - cmake
-    - ninja
```
Comment (jakirkham) [inline recipe/meta.yaml]: Shouldn't we keep this if we want to test it with `cotire`?

Comment (jschueller) [inline recipe/meta.yaml]: @jakirkham, I can leave it if you want to test, I could not make it work properly

**Comment by jakirkham** [comment]
Please see issue ( https://github.com/conda-forge/conda-forge.github.io/issues/147 ). Basically we haven't formalized a way to do dev releases. We should probably try to do that first. Otherwise users may be forced into an rc or dev release when they want stable releases.

**Comment by jschueller** [comment]
@jakirkham, thank you for the pointer.
In the case of OpenTURNS we always push rc's as they're almost stable and bring lots of bug fixes, also the rc period is rather short (1 month or so).

**Comment by jakirkham** [comment]
Still we should have a proper way of doing RCs that don't fall in the `main` label. It would be nice if we could resolve that first and then proceed.

**Pull request merged**